include/sender_cadmium.hpp

```diff
@@ -177,7 +181,7 @@ class Sender {
         }
 
         friend std::ostringstream& operator<<(std::ostringstream& os,
-		                   const typename Sender<TIME>::state_type& i) {
+            const typename Sender<TIME>::state_type& i) {
             os << "packetNum: " << i.packet_num << 
```

Comment:
My suggestion would be not to give any space before and after << operator.
i.e., os<<"packetNum: "<<i.packet_num;

Reply:
I think spaces are OK around << characters.